src/Sarif.Viewer.VisualStudio/Tags/ISarifLocationTagger.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.VisualStudio.Text.Tagging;
+using Microsoft.VisualStudio.TextManager.Interop;
+using System;
+
+namespace Microsoft.Sarif.Viewer.Tags
+{
+    internal interface ISarifLocationTagger
+    {
+        /// <summary>
+        /// Adds a tag to report to visual studio.
+        /// </summary>
+        /// <param name="sourceRegion">The original span from the region in the SARIF log.</param>
+        /// <param name="documentSpan">The span to use to create the tag relative to an open document.</param>
+        /// <param name="runId">The SARIF log run identifier associated with this tag.</param>
+        /// <param name="textMarkerTag">The text marker tag to display for this tag.</param>
+        /// <returns>Returns a new instance of <see cref="ISarifLocationTag"/></returns>
+        /// <remarks>
+        /// This <paramref name="documentSpan"/>is not necessarily the same as <paramref name="documentSpan"/>.
```

Comment:
sp

Reply:
What exactly? Spell checker seems happy.

---
In reply to: [481412872](https://github.com/microsoft/sarif-visualstudio-extension/pull/191#discussion_r481412872) [](ancestors = 481412872)